### DISCUSSION TITLE
request: compact mode --- no content loaded.

### DISCUSSION THREAD

**Discussion by OSStuntman** [jocmp/capyreader#601] [Feature Requests]
Hello,

I'd like a compact mode option added:

Article name, date, and author.

**Comment by issteve** [comment]
I would like to have a lot of articles on the first view/page without the need to scroll to much.
So it would prefer a compact mode even shorter with just the title/headline and the date.
(And the possibility to adjust the font size just for the front page, too.)

The standard view of SparseRSSMod is a good example:
![Compact-RSS-View](https://github.com/user-attachments/assets/c52710fb-6909-435c-a17b-994f6d646f10)

**Reply by jocmp** [reply]
hey @OSStuntman there are a few settings for this currently under Settings > Display & Appearance. Here what it looks like with feed name, icon and images all turned off. Let me know what you think!

| Settings | Feed |
|---|---|
| ![Screenshot_20250105_141543](https://github.com/user-attachments/assets/d9714ba8-d890-4de4-bc7a-9b419e86478b) | ![Screenshot_20250105_141557](https://github.com/user-attachments/assets/1cdbde16-4a75-4650-9149-d232cc996204) |

**Reply by issteve** [reply]
Wow, thank you very much for your fast reply! 
Even you didn't directly address me, yes, thank you, for me, this is ok, so far. 
I just would like to have way less space between the article headlines and a real date with time there (instead of just e.g. "3h"). 

(And less space between the feeds in the feeds overview, too btw. 
Somehow the font size slider is not very helpful on my phone with the first step being a little bit to small and the next one being to big, too... ;-)

**Reply by jocmp** [reply]
@issteve I'm tracking a change (https://github.com/jocmp/capyreader/issues/681) to show real dates. I'm also looking into reworking the list font slider (https://github.com/jocmp/capyreader/pull/683). I added it once and haven't revisited it since. Stay tuned!

**Comment by OSStuntman** [comment]
Thanks for the reply.

screenshots:
https://drive.proton.me/urls/WTY12VQQ5W#5PFci52mZ6WM

Even with these setting, the app is retrieving article content.

I don't know if it is actually possible to not retrieve article content or if I'm asking you for an illusion.

--- .\.\aurice

-------- Original Message --------

**Reply by jocmp** [reply]
@OSStuntman it would be difficult to rework the app to exclude app content. Just wondering, why do you want to view just the title in the reader view?

**Comment by OSStuntman** [comment]
@jocmp,

I have 115 feeds, and they are all pulling whatever is in the content, which often includes images.

I mainly scan the article titles, check the time/date the article was posted and by whom.

--- .\.\aurice

-------- Original Message --------
